packages/waku/src/cli.ts

```diff
@@ -144,6 +145,7 @@ async function runStart() {
     import(pathToFileURL(path.resolve(distDir, DIST_ENTRIES_JS)).toString());
   const app = new Hono();
   app.use(contextStorage());
+  app.use(compress())
```

Comment:
It looks like a good addition, it's not 100% certain if this won't cause any issues for all users.
So, my suggestion is to introduce a new option `--experimental-compress` and use it explicitly.
We can also enable it for the dev server too.

Reply:
Done